intro-to-git-everyone/user_info/Ethan.md

```diff
@@ -0,0 +1,6 @@
+# Ethan Graybar
```

Comment:
This file shouldn't be here. I think this is from when you had checked out a copy of the repository inside another copy of the repository. We definitely don't want to commit that. So I would remove this file before merging in this pull request.